docs/tutorials/hub/user-thread-database.md

```diff
@@ -0,0 +1,194 @@
+# Thread Database
+
+In this tutorial, we'll walk through using the local Thread Database in JavaScript. The `Database` can be used to store and work with data entirely locally in your app and it can be linked to other nodes that can help sync, rely, or store data when the user goes offline. We'll use those features to create a Database, connect it to the Hub, and connect two peers.
+
+## Getting Started
+
+There are a few resources you'll need before you start writing code.
+
+- [A new Typescript web app](https://webpack.js.org/guides/typescript/). We recommend using Typescript, as Textile libraries are in a period rapid of development and type detection is valuable during upgrades.
+- You should already be familiar with how to create user identities and how to generate API keys.
+
+## Create a new Database
+
+To create a new `Database` you need to, 
+
+1. Import the `Database` class.
+2. Create a new database object with local storage enabled.
+3. Attach that database to your Hub account for networking.
+
+To attach you Hub account, you'll use the non-signing key described in the [development mode](development-mode.md) section. You can pass it in using the `withKeyInfo` method you've already used on the `Buckets` class.
+
+Let's take a look at those steps all together.
+
+```typescript
+import { KeyInfo, ThreadID } from '@textile/hub'
+import { Database } from "@textile/threads-database"
+import LevelDatastore from "datastore-level"
+
+const init = (keyInfo: KeyInfo, threadID: ThreadID) => {
+    const storage = new LevelDatastore(threadID.toString())
+    const db = Database.withKeyInfo(keyInfo, storage)
```

Comment:
```suggestion
const db = Database.withKeyInfo(keyInfo, threadID.toString())
```

Reply:
I've made it so they don't need to worry about actually creating the storage object. It'll just use the string as the db name.

Reply:
very nice